core/services/src/main/java/org/exoplatform/services/wcm/portal/impl/LivePortalManagerServiceImpl.java

```diff
@@ -193,7 +197,14 @@ private Node getLivePortalsStorage(final SessionProvider sessionProvider) throws
     String portalsStoragePath = locationEntry.getPath();
     ManageableRepository manageableRepository = repositoryService.getCurrentRepository();
     Session session = sessionProvider.getSession(workspace, manageableRepository);
-    return (Node)session.getItem(portalsStoragePath);
+    try {
+      return (Node) session.getItem(portalsStoragePath);
+    } catch (PathNotFoundException e) {
+      if (SITES_PATH.equals(portalsStoragePath)) {

```

Comment:
What about starting the NodeHierarchyCreator here ?  could it solve the issue ?

Reply:
it is not possible!